native/Gopkg.toml

```diff
@@ -0,0 +1,30 @@
+# Gopkg.toml example
```

Comment:
I'm a bit surprised to see this file in Java-based native driver folder :)

Reply:
I wonder WTF does that there too :1st_place_medal: